### PR TITLE
added x-ms-mutability for sku

### PR DESCRIPTION
**Pull request by archanac866**
This is to fix [OBJECT_ADDITIONAL_PROPERTIES](https://github.com/Azure/oav/blob/master/documentation/oav-errors-reference.md#OBJECT_ADDITIONAL_PROPERTIES) for sku field. [link](https://portal.azure-devex-tools.com/tools/api-live-validation/errors?apiVersion=2020-05-12-preview&commitHash=d5ff2c358382dfa75282bf3c13aa00dd92c38d71&endDate=2022-11-07&operationId=FarmBeatsModels_Get&resourceProvider=microsoft.agfoodplatform&specsRepo=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs&startDate=2022-11-01&validationId=ARM&validatorName=default-early)

Let me know if anything else is needed. Would want to close this ASAP so a s to close SDK signoff we need for public preview PLR criteria.

Reason for this change:
	
This property has already been there since day 1. It's not a new property that we are newly introducing. We chose to not have it in swagger so that always default one is used. Because of the live validation error, we need to add it now which if not done it would raise issues in getting management plane SDK signoff needed for public PLR. hence the ask.

### CODE REVIEW

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json
Comment (jianyexi) [inline specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:1111]: this $ref seems incorrect ?

**Comment by priyankajayaswal1** [comment]
This change has been from day 1. Hence this is more of a cosmetic change than a new api version. Need suggestions on how to proceed. Looking forward to approval to start the SDK signoff process.

**Comment by priyankajayaswal1** [comment]
Created breaking change review: https://msazure.visualstudio.com/One/_workitems/edit/16198030
Waiting for approval.

**Comment by priyankajayaswal1** [comment]
I have sent a mail to get api review ASAP titled Breaking Change Approval for sku field

**Comment by mikekistler** [comment]
I added the Approved-BreakingChange label since this change is allowed prior to public preview.

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json
Comment (mikekistler) [inline specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:1137]: It looks like this schema is only used as the request body for patch, which is documented as "Update" for the FarmBeats resource. Can this patch operation also create a FarmBeats resource? If not, then there is no need -- and more likely confusing -- to add "sku" to the request model here with "x-ms-mutability: [read, create]".

Said another way -- this schema is only used for update, so a property only allowed for create has no purpose here.

Comment (priyankajayaswal1) [reply specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json]: removed as suggested

**Review by jianyexi — APPROVED** [review]

**Pull request merged**
